meson.build

```diff
@@ -267,6 +267,10 @@ flags = ['-D_FILE_OFFSET_BITS=64']
 link_flags = []
 
 test_flags = ['-Werror=implicit-function-declaration',
+              '-Wno-missing-field-initializers',
+              '-Wno-sign-compare',
+              '-Wno-unused-parameter',
+              '-Wno-cast-function-type',
```

Comment:
Those two could be also fixed, I suppose one day.

Reply:
> The second is required for the FARPROC casting thing, so the only way to remove that is to cast functions to void * first, which just moves the problem somewhere else.

Where? FARPROC can go through void*, nothing wrong about it.

> The first one requires us writing something like MP_UNUSED() for every unused parameters. I feel this increases boilerplate.

Ah, not MP_UNUSED is not nice, but maybe we can just remove unused parameters.

Reply:
Ok, I guess, drilling into warnings often makes you workaround warning than actual code